src/robot.coffee

```diff
@@ -393,26 +407,6 @@ class Robot
         scriptDocumentation.commands.push cleanedLine
         @commands.push cleanedLine
 
-  # Public: A helper send function which delegates to the adapter's send
-  # function.
-  #
-  # user    - A User instance.
-  # strings - One or more Strings for each message to send.
-  #
-  # Returns nothing.
-  send: (user, strings...) ->
-    @adapter.send user, strings...
```

Comment:
Hmm, the uses send an envelope instead of a user, so its just a docs issue. I need to consolidate routes to the adapter to insert hooks, though; so I will:

 * update docs
 * send response to here
 * move the logic I added to response here